pkg/minikube/cluster/cluster_linux.go

```diff
@@ -21,7 +21,10 @@ import (
 	"path/filepath"
 
 	kvm "github.com/dhiltgen/docker-machine-kvm"
+	local "github.com/dims/docker-machine-local"
```

Comment:
Whats the difference between your driver and libmachine's?

Either way, we should have the driver code in minikube